module.js

```diff
@@ -16,7 +16,9 @@ var hook = function(req, res) {
 
 var github = function(payload, localToken) {
     // TODO: Only acknowledge pushes to the "Master" branch.
-    console.log("Repo: " + payload.repository.full_name);
+    console.log('Repo: ' + payload.repository.full_name);
+    console.log('Payload: ');
```

Comment:
Looks like we don't need a trailing space, since the `:` will be the last readable character printed on that line.